contracts/interfaces/IEntryPoint.sol

```diff
@@ -144,6 +144,13 @@ interface IEntryPoint is IStakeManager, INonceManager {
         bytes signature;
     }
 
+    /**
+     * Get the EntryPoint contract's version.
+     *
+     * @return the ERC-712 value used as DOMAIN_VERSION for UserOperation hash and signature calculation.
```

Comment:
please change the version to "1" - 
current value "0.8" is not a valid ERC-712 domain (should be the "major version").
also, it is the version of the ABI, not the current implementation version.